dotnet/src/Skills/Skills.OpenAPI/Extensions/KernelChatGptPluginExtensions.cs

```diff
@@ -24,6 +26,34 @@ namespace Microsoft.SemanticKernel;
 /// </summary>
 public static class KernelChatGptPluginExtensions
 {
+    public static async Task<IDictionary<string, ISKFunction>?> ImportPluginManifestFromUrlAsync(
```

Comment:
This method reads/parses a custom plugin registry JSON document that is not part of the ChatGPT/OpenAI plugin specification, as far as I know. Therefore, I propose moving it out from the KernelChatGptPluginExtensions class to another class, such as Kernel**Custom**PluginExtensions or any other class that will group similar functionality together and provide better context.

Reply:
Yeah agreed! I whipped this up quickly to start getting just this kind of feedback. Thanks!